mmv1/products/compute/ResizeRequest.yaml

```diff
@@ -0,0 +1,168 @@
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_delete: templates/terraform/custom_delete/compute_mig_resize_request_delete.go.erb
+
+# If true, code for handling long-running operations is generated along with
+# the resource. If false, that code is not generated.
+autogen_async: true
+# Sets parameters for handling operations returned by the API.
+async: !ruby/object:Api::OpAsync
+  # Overrides which API calls return operations. Default: ['create',
+  # 'update', 'delete']
+  # actions: ['create', 'update', 'delete']
+  operation: !ruby/object:Api::OpAsync::Operation
+    kind: "compute#operation"
+    path: "name"
+    base_url: "projects/{{project}}/zones/{{zone}}/operations/{{op_id}}"
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: "targetLink"
+  status: !ruby/object:Api::OpAsync::Status
+    path: "status"
+    complete: "DONE"
+    allowed:
+      - "PENDING"
+      - "RUNNING"
+      - "DONE"
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error/errors"
+    message: "message"
+
+# Resize request parameters injected via URL
+parameters:
+  - !ruby/object:Api::Type::ResourceRef
+    name: "zone"
+    resource: "Zone"
+    imports: "name"
+    description: |
+      Name of the compute zone scoping this request. Name should conform to RFC1035.
+    required: true
+    url_param_only: true
+  - !ruby/object:Api::Type::ResourceRef
+    name: "instanceGroupManager"
+    resource: "InstanceGroupManager"
+    imports: "name"
+    description: |
+      The name of the managed instance group. The name should conform to RFC1035 or be a resource ID.
+
+      Authorization requires the following IAM permission on the specified resource instanceGroupManager:
+      *compute.instanceGroupManagers.update
+    required: true
+    url_param_only: true
+  - !ruby/object:Api::Type::ResourceRef
+    name: "resizeRequest"
+    resource: "ResizeRequest"
+    imports: "name"
+    description: |
+      The name of an existing resize request. The name should conform to RFC1035 or be a resource ID.
+    url_param_only: true
+
+# Resize request parameters sent in the body of the CRUD action
+properties:
+  - !ruby/object:Api::Type::Time
+    name: "creationTimestamp"
+    description: |
+      The creation timestamp for this resize request in RFC3339 text format.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "name"
+    description: |
+      The name of this resize request. The name must be 1-63 characters long, and comply with RFC1035.
+    required: true
+  - !ruby/object:Api::Type::String
+    name: "description"
+    description: |
+      An optional description of this resize-request.
+  - !ruby/object:Api::Type::String
+    name: "zone"
+    description: |
+      The URL of a zone where the resize request is located. Populated only for zonal resize requests.
+    output: true
+  - !ruby/object:Api::Type::Integer
```

Comment:
can we remove the deprecated parameter? I would rather not introduce this, so that we don't have to delete it as a part of breaking change in the future...